### PR TITLE
Enabled interoperability with Rails 5.0.x...

### PR DESCRIPTION
**Pull request by mvastola**
Created relevant Appraisal and configured Travis CI testing for Rails 5.0

All CI tests passing with all Rails versions.

Resolves issue #307.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ rvm:
   - 1.9
   - 2.0
   - 2.1
-  - 2.2
+  - 2.2.2
```
Comment (rwz) [inline .travis.yml]: why change this?

Comment (mvastola) [inline .travis.yml]: Rails 5.0.x has a minimum ruby version requirement of 2.2.2. Specifying just `2.2` to Travis resolves as version 2.2.0 and causes bundler to fail.

Comment (rwz) [inline .travis.yml]: That's weird, it should resolve to the latest 2.2.x according to their specs.

Comment (mvastola) [inline .travis.yml]: It's an RVM thing, I believe.. see [here](https://github.com/rvm/rvm/blob/stable/config/known#L10). Specifying just `2.2` resolves to `2.2.1` (I was apparently mistaken above when I said `2.2.0`) in the latest stable release of RVM. (It seems the revision number will jump on the next RVM release as this is different in RVM's `master` branch.)

In any case, I assure you it's necessary. I only changed it after it didn't work.

Comment (mvastola) [inline .travis.yml]: Also see https://travis-ci.org/rails/jbuilder/jobs/83196945#L90, the latest CI test of `jbuilder` `master` branch where the RVM ruby version was specified as `2.2`.

**Comment by rwz** [comment]
Thanks for the PR. LGTM.

I've [tweeted at TravisCI](https://twitter.com/rwz/status/681017712307380224), let's see if we could make them fix the ruby 2.2 problem before merging this.

**Comment by mvastola** [comment]
Ok. 

FYI though, looking at https://travis-ci.org/rails/jbuilder/jobs/83196945#L86, it would seem that the issue isn't really with Travis since Travis is passing the version info correctly to RVM. And then RVM is simply interpreting the revision number as documented. (If you were to run `rvm list known` the then-current release of RVM would have had the entry `[ruby-]2.2[.0]`.)  

If you want to work around this, you can add this to `.travis.yml` (since, as I pointed out, its `master` branch, has `2.2` resolving to `2.2.4`), but note that you will then be relying on a dev version of RVM.
```
before_script:
  - rvm get master
```

The alternative is to just wait for the next version of RVM to come out and then you should be able to switch `2.2.2` back to `2.2` in `.travis.yml`.

I'd be surprised if Travis tweets you back anything to the contrary, but can't hurt to see what they have to say, I suppose..

Do you happen to recall where in the Travis docs you saw the behavior specified that the latest version should be pulled?

**Comment by rwz** [comment]
> Do you happen to recall where in the Travis docs you saw the behavior specified that the latest revision of ruby matching the major/minor version specified would be used?

Nope.

I guess you're right, my latest stable RVM points 2.2 to 2.2.1, which is better than 2.2.0 but still not good enough.

Sigh, let me try the `rvm get master` hack.

**Comment by rwz** [comment]
Alright, the `get master` hack didn't work. I've specified full versions for 2.2 and 2.3 and it [seems to be working fine](https://travis-ci.org/rails/jbuilder/builds/98948068). Could you please rebase your PR against master so I could merge it?

**Comment by mvastola** [comment]
Hmm. Yea. It looks like `before_script` doesn't run until way after the ruby is installed.
Rebasing now.

**Comment by rwz** [comment]
Awesome, thanks

**Comment by mvastola** [comment]
Thank you as well!

**Comment by rwz** [comment]
Released 2.4.0

**Pull request merged**